### ISSUE TITLE
[Webhook connector] Clarify the format we use to access request body

### ISSUE THREAD

**Issue by markfarkas-camunda** [camunda/connectors#2564]
__What should we do?__

Clarify what is the correct format of the Correlation key, and what is the exact way to extract info from the request body. (`request.body.name`)

We already have a documentation but it would be nice to have a placeholder value in the input field, or a link to the doc in the description.

__Why should we do it?__

It is not obvious for first time users what is the correct keyword (request) to access the body of the request. They might assume it is just `body.name` or `req.body.name` instead of `request.body.name`

**Comment by johnBgood** [comment]
@crobbins215 we discussed this during the architecture session, and what @markfarkas-camunda pointed out is that we don't have a consistent naming so far.
For the [Kafka connector](https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/kafka/#make-your-kafka-consumer-connector-executable) the key would be `value.myKey` for instance, whereas for the [Webhook connector](https://docs.camunda.io/docs/components/connectors/protocol/http-webhook/#make-your-http-webhook-connector-executable) it'd be `request.myKey`.

We discussed possibly using a common term (`message`, `inputData`,...) but this is not ideal as the term is tightly coupled to the domain object (Webhook <-> request, Kafka <-> value) and the user would still need to look at the documentation to find inner fields (`request.body` for instance).

Maybe we should solve this with better documentation + nice tooltips in the modeler?

**Comment by crobbins215** [comment]
Could we improve the documentation for this as part of this ticket then?

**Comment by sbuettner** [comment]
Agreed on improving the documentation first and find a way for a uniform message property model later.

**Comment by crobbins215** [comment]
Closing this epic for now, as it's not currently prioritised or actionable based on the insights available. If new context or customer needs emerge, we can revisit and reopen it as needed.
